packages/mask/src/plugins/Avatar/assets/verify2.tsx

```diff
@@ -0,0 +1,19 @@
+import { createIcon } from '@masknet/icons'
+export const Verify2Icon = createIcon(
```

Comment:
The `.tsx` files should use the big camel case. Please rename this file to `Verfiy2.tsx`.